frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -1038,28 +1038,54 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
 
 export const CSVSubscription: React.FC<CSVSubscriptionProps> = ({
   obj,
-  subscriptions = [],
+  customData: { subscription },
   ...rest
 }) => {
   const EmptyMsg = () => (
     <MsgBox title="No Operator Subscription" detail="This Operator will not receive updates." />
   );
 
-  const subscription = React.useMemo(() => subscriptionForCSV(subscriptions, obj), [
-    obj,
-    subscriptions,
-  ]);
-
   return (
     <StatusBox EmptyMsg={EmptyMsg} loaded data={subscription}>
       <SubscriptionDetails {...rest} obj={subscription} clusterServiceVersions={[obj]} />
     </StatusBox>
   );
 };
 
+/**
+ * Give the name of the operator from subscription to the CSV Details Page
+ */
+export const CSVDetailsPageWrapper: React.FC<CSVDetailsPageWrapper> = (props) => {
+  const { match } = props;
+  const subscriptionResource = React.useMemo(
```

Comment:
I wouldn't expect us to need a new watch. We already have a Subscription tab, so I'd expect us to already have the data somewhere.

Reply:
The whole idea of this PR is to get the subscription before getting package manifests so that we can narrow down the scope of our request for it. I have removed the block for subsequent fetching of Subscription and supplying this instance.
 https://github.com/openshift/console/pull/7061/files/a52ebe7ec2f3dd0c51bba749c16b8d100241665d#diff-c7de8d95faa26753bd35817bb04d94993b4f37ecd076de5edd8690488d8c8ecfL1141.